### ISSUE TITLE
Better motivation...

### ISSUE THREAD

**Issue by davidrpugh** [ScalABM/requirements#1]
I am not sure that we should motivate ScalABM (or ABM in general) by setting it up as a direct competitor to DSGE models.  I think this is the approach that ABM modelers have taken in the past and I think it has failed to persuade outside researchers (i.e., non-ABM converts); instead we should push ABM as a complementary approach to DSGE models that is better placed to leverage increasingly available large-scale micro panel data sets.  

Proponents of DSGE modelers love to talk about "structural modeling" and "microfoundations"; I think it is important to stress that ABM models are also micro-founded, structural models.  A major difference, as I see it, is that while DSGE models impose structure on preferences and production technologies, ABMs impose structure in the form of contractual relationships, market institutional structures, government policy, etc. The structure of preferences and production technologies are inherently difficult to observe, we can, however, easily observe contracts, market structures, government policies, etc and incorporate this structure into our ABMs.

I also think we should place estimation, calibration, and validation of models using ScalABM front and center as part of the motivation.  Historically, this has been a weak spot of macroeconomic ABM models. 
Basically: ScalABM combines "Big Data" techniques, particularly machine learning methods, that are capable of taking advantage of large-scale micro panel data sets with structural economic modeling.

**Comment by prauwolf** [comment]
This all sounds good to me, I'll make the change.

**Comment by prauwolf** [comment]
@davidrpugh I made changes to the "Motivation" as well as the "Solution: ScalABM" sections in an attempt to avoid motivating ScalABM with limitations to DSGE models. Further, I've included more explicit references to ScalABM's intention to provide calibration and validation techniques. What do you think?
